app-i18n/jfbterm/jfbterm-0.4.7-r5.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools font
+
+DESCRIPTION="The J Framebuffer Terminal/Multilingual Enhancement with UTF-8 support"
+HOMEPAGE="https://osdn.net/projects/jfbterm/"
+SRC_URI="mirror://sourceforge.jp/${PN}/13501/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="debug"
+
+DEPEND="sys-libs/ncurses:0"
```

Comment:
This should be := instead, and it should be in RDEPEND too, so RDEPEND="${DEPEND}", if ncurses is linked against...

Reply:
It's not linked against ncurses. I found no mention of curses.h or -lcurses in code or build process.
There's terminfo file, so it's absolutely RDEPEND, but not DEPEND.

Reply:
Huh! Please add a comment indicating that, in that case.

Reply:
There's also runtime dependency on gunzip for unpacking compressed font.